src/core/components/responses.jsx

```diff
@@ -162,6 +168,20 @@ export default class Responses extends React.Component {
               }
             </tbody>
           </table>
+          {/* OutSystems change: Call the headers logic here instead of using inside the response.jsx component in order to have a new table below the body response */}
+          {
+            responses.entrySeq().map(([code, response]) => {
+
+              let headers = response.get("headers")
+              return (
+                <Headers
+                  headers={headers}
```

Comment:
Elements in an array should have an unique key property.

Reply:
good tip. thank you

Reply:
done